tools/acorn-optimizer.js

```diff
@@ -584,6 +584,28 @@ function isDynamicDynCall(node) {
   );
 }
 
+function isExportWrapperFunction(f) {
+  let found;
+  fullWalk(f.body, inner => {
```

Comment:
A full walk here doesn't seem right. We don't want to recursively look for such a return, but we want the entire body to be one, I think?

Reply:
Yes I guess so.. I think I was trying be flexible in case we add other statements before the return, but I guess we want to be as precise as possible here to avoid false positives.

I will adjust accordingly.